pkg/acceptance/bettertestspoc/assert/resourceshowoutputassert/secret_show_output_gen.go

```diff
@@ -0,0 +1,89 @@
+// Code generated by assertions generator; DO NOT EDIT.
+
+package resourceshowoutputassert
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/bettertestspoc/assert"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+)
+
+// to ensure sdk package is used
+var _ = sdk.Object{}
+
+type SecretShowOutputAssert struct {
+	*assert.ResourceAssert
+}
+
+func SecretShowOutput(t *testing.T, name string) *SecretShowOutputAssert {
+	t.Helper()
+
+	s := SecretShowOutputAssert{
+		ResourceAssert: assert.NewResourceAssert(name, "show_output"),
+	}
+	s.AddAssertion(assert.ValueSet("show_output.#", "1"))
+	return &s
+}
+
+func ImportedSecretShowOutput(t *testing.T, id string) *SecretShowOutputAssert {
+	t.Helper()
+
+	s := SecretShowOutputAssert{
+		ResourceAssert: assert.NewImportedResourceAssert(id, "show_output"),
+	}
+	s.AddAssertion(assert.ValueSet("show_output.#", "1"))
+	return &s
+}
+
+////////////////////////////
+// Attribute value checks //
+////////////////////////////
+
+func (s *SecretShowOutputAssert) HasCreatedOn(expected time.Time) *SecretShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("created_on", expected.String()))
+	return s
+}
+
+func (s *SecretShowOutputAssert) HasName(expected string) *SecretShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("name", expected))
+	return s
+}
+
+func (s *SecretShowOutputAssert) HasSchemaName(expected string) *SecretShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("schema_name", expected))
+	return s
+}
+
+func (s *SecretShowOutputAssert) HasDatabaseName(expected string) *SecretShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("database_name", expected))
+	return s
+}
+
+func (s *SecretShowOutputAssert) HasOwner(expected string) *SecretShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("owner", expected))
+	return s
+}
+
+func (s *SecretShowOutputAssert) HasComment(expected string) *SecretShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("comment", expected))
+	return s
+}
+
+func (s *SecretShowOutputAssert) HasSecretType(expected string) *SecretShowOutputAssert {
+	s.AddAssertion(assert.ResourceShowOutputValueSet("secret_type", expected))
+	return s
+}
+
+func (s *SecretShowOutputAssert) HasOauthScopes(expected []string) *SecretShowOutputAssert {
```

Comment:
is this assertion generated or altered manually?

Reply:
I've added it manually to check if this would work. It didn't so it is to be removed

Reply:
Please add such additions to _ext.go files (close after reading)